File: Service_eHospital_DongNai_A/Implement/DuocGiaVonService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class DuocGiaVonService : BaseService<DuocGiaVon, IDuocGiaVonRepository>
    , IDuocGiaVonService
    {
    private readonly IDuocGiaVonRepository _DuocGiaVonRepository;
    public DuocGiaVonService(IDuocGiaVonRepository DuocGiaVonRepository) : base(DuocGiaVonRepository)
    {
    _DuocGiaVonRepository = DuocGiaVonRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A/DuocGiaVon.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DuocGiaVon } from './DuocGiaVon.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DuocGiaVonService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'DuocGiaVon_Id', 'DuocKyTonKho_Id', 'Duoc_Id', 'DonGiaVon', 'NgayTao', 'NguoiTao_Id', 'NgayCapNhat', 'NguoiCapNhat_Id'];

    List: DuocGiaVon[] | undefined;
    ListFilter: DuocGiaVon[] | undefined;
    FormData!: DuocGiaVon;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DuocGiaVon";
    }
}

